Library/Homebrew/linkage_checker.rb

```diff
@@ -112,7 +112,7 @@ def check_dylibs(rebuild_cache:)
         # weakly loaded dylibs may not actually exist on disk, so skip them
         # when checking for broken linkage
         keg_files_dylibs[file] =
-          file.dynamically_linked_libraries(except: :LC_LOAD_WEAK_DYLIB)
+          file.dynamically_linked_libraries(except: :DYLIB_USE_WEAK_LINK)
```

Comment:
Should we not exclude both?

Reply:
I put a compatibility layer on the ruby-macho side:

https://github.com/Homebrew/ruby-macho/blob/af38cc83941ab0c5e91a8dd99f6a598a13f31d85/lib/macho/load_commands.rb#L558-L559